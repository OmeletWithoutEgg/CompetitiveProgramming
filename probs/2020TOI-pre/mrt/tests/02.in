69
37 9 11 1 24 36 36
23 3 9 9 11 21 15
5 0 1 4 0 0 1
43 42 30 15 6 36 34
13 0 2 9 9 5 6
3 1 0 2 2 0 0
13 4 9 7 9 1 12
5 3 0 4 4 2 3
11 1 5 10 5 0 4
11 5 3 8 4 0 6
43 39 14 27 29 28 15
7 6 1 0 6 3 2
41 33 38 6 1 19 12
5 1 1 3 1 4 3
31 21 24 12 19 30 8
3 2 1 1 1 2 2
23 0 1 13 6 15 8
13 1 4 2 1 5 11
53 41 5 21 51 42 5
11 9 4 0 2 8 4
17 4 7 14 6 6 3
37 6 12 22 5 30 21
43 13 7 28 7 18 28
7 3 5 4 5 1 3
47 21 45 11 44 21 21
17 16 16 1 13 4 5
3 2 1 0 1 2 1
17 6 8 1 10 9 14
37 1 29 17 1 10 22
13 7 12 7 1 10 9
47 35 26 30 39 7 35
47 17 39 44 14 11 37
59 23 37 20 1 33 25
23 12 19 6 10 18 15
13 3 6 3 9 8 3
17 4 2 0 11 8 6
19 1 15 7 17 15 2
43 18 33 24 23 37 15
17 8 1 5 8 11 14
11 1 7 5 4 1 3
31 7 24 23 18 9 17
13 2 2 4 10 12 5
17 5 10 9 11 14 11
41 28 21 37 3 9 31
7 4 6 6 6 4 3
19 13 11 1 14 2 11
5 0 3 1 3 2 1
3 2 1 1 2 2 2
59 14 3 53 23 4 51
17 3 12 15 10 13 2
7 3 4 1 6 2 2
31 8 14 10 28 12 3
17 13 2 2 11 0 6
23 10 4 12 9 17 2
41 38 30 15 11 36 17
47 21 34 40 25 33 45
7 5 1 1 0 4 6
19 9 11 0 12 18 1
2 1 1 1 1 0 1
17 13 4 15 15 6 3
11 4 9 9 9 3 2
29 12 26 18 10 28 19
7 2 0 4 3 0 0
41 32 4 22 9 1 18
19 13 18 13 3 4 1
23 8 1 2 18 16 4
2 1 1 1 1 0 1
13 2 5 6 5 4 5
2 0 1 1 0 0 1
