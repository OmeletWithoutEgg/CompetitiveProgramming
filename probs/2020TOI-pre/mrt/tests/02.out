36 36
11 4
2 3
32 0
2 7
2 2
2 5
3 1
2 2
1 3
12 24
5 4
32 23
2 4
22 6
1 1
19 12
7 9
49 25
1 5
16 11
26 13
7 21
4 5
17 2
4 5
1 1
3 11
19 32
6 7
27 16
17 20
52 51
7 20
5 4
7 13
2 5
18 30
2 5
3 9
27 5
11 4
11 11
37 3
1 2
6 18
1 3
0 2
24 48
5 4
6 0
18 21
0 6
7 16
11 4
33 45
6 3
8 5
1 1
11 4
8 6
16 14
4 3
14 30
18 2
6 14
1 1
3 5
0 1
